sdk/monitor/Azure.Monitor.Query/tests/LogsTestData.cs

```diff
@@ -4,10 +4,12 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading;
 using System.Threading.Tasks;
 using Azure.Core;
 using Azure.Core.TestFramework;
 using Azure.Monitor.Query;
+using NUnit.Framework;
```

Comment:
The using changes can be reverted because there are no other changes in this file.